src/frontend/hooks/useGetQuest.ts

```diff
@@ -0,0 +1,26 @@
+import { useQuery, useQueryClient } from 'react-query'
+
+export default function useGetQuest(questId: number | null) {
+  const queryClient = useQueryClient()
+  const queryKey = `getQuest:${questId}`
+  const query = useQuery(
+    queryKey,
+    async () => {
+      if (questId === null) {
+        return {}
+      }
+      const response = await window.api.getQuest(questId)
+      if (!response) return null
+      return response
```

Comment:
if you add a type casting here you can remove the `ts-ignores` in the QuestViewer components

```ts
return response as {
    name: string
    capsule_image: string
}
```

Reply:
Good idea. I think you meant to add this comment to the `useGetSteamGame` hook. I added it there. I also made both fields optional as technically this is an `unknown` type since we don't have runtime type checking on the http response